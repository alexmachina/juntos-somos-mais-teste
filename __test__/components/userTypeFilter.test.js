import React from "react";
import UserTypeFilter from "../../pages/customers/list/components/filters";
import renderer from "react-test-renderer";

describe("Components", () => {
  describe("UserTypeFilter", () => {
    it("Renders correctly", () => {
      const tree = renderer.create(<UserTypeFilter />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
