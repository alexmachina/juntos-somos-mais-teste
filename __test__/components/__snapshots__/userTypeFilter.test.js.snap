// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components UserTypeFilter Renders correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation1 makeStyles-filters-104 MuiPaper-rounded"
    >
      <div>
        <header>
          <h6
            className="MuiTypography-root MuiTypography-h6"
          >
            Usuário
          </h6>
        </header>
        <article>
          <div
            className="MuiFormGroup-root"
          >
            <label
              className="MuiFormControlLabel-root"
            >
              <span
                aria-disabled={false}
                className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-176 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={null}
              >
                <span
                  className="MuiIconButton-label"
                >
                  <input
                    className="PrivateSwitchBase-input-179"
                    data-indeterminate={false}
                    onChange={[Function]}
                    type="checkbox"
                    value="special"
                  />
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                </span>
              </span>
              <span
                className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                Especial
              </span>
            </label>
            <label
              className="MuiFormControlLabel-root"
            >
              <span
                aria-disabled={false}
                className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-176 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={null}
              >
                <span
                  className="MuiIconButton-label"
                >
                  <input
                    className="PrivateSwitchBase-input-179"
                    data-indeterminate={false}
                    onChange={[Function]}
                    type="checkbox"
                    value="normal"
                  />
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                </span>
              </span>
              <span
                className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                Normal
              </span>
            </label>
          </div>
        </article>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation1 makeStyles-filters-201 MuiPaper-rounded"
    >
      <div>
        <header>
          <h6
            className="MuiTypography-root MuiTypography-h6"
          >
            Região
          </h6>
        </header>
        <article>
          <div
            className="MuiFormGroup-root"
          >
            <label
              className="MuiFormControlLabel-root"
            >
              <span
                aria-disabled={false}
                className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-176 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={null}
              >
                <span
                  className="MuiIconButton-label"
                >
                  <input
                    className="PrivateSwitchBase-input-179"
                    data-indeterminate={false}
                    onChange={[Function]}
                    type="checkbox"
                    value="norte"
                  />
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                </span>
              </span>
              <span
                className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                Norte
              </span>
            </label>
            <label
              className="MuiFormControlLabel-root"
            >
              <span
                aria-disabled={false}
                className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-176 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={null}
              >
                <span
                  className="MuiIconButton-label"
                >
                  <input
                    className="PrivateSwitchBase-input-179"
                    data-indeterminate={false}
                    onChange={[Function]}
                    type="checkbox"
                    value="nordeste"
                  />
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                </span>
              </span>
              <span
                className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                Nordeste
              </span>
            </label>
            <label
              className="MuiFormControlLabel-root"
            >
              <span
                aria-disabled={false}
                className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-176 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={null}
              >
                <span
                  className="MuiIconButton-label"
                >
                  <input
                    className="PrivateSwitchBase-input-179"
                    data-indeterminate={false}
                    onChange={[Function]}
                    type="checkbox"
                    value="centro-oeste"
                  />
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                </span>
              </span>
              <span
                className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                Centro Oeste
              </span>
            </label>
            <label
              className="MuiFormControlLabel-root"
            >
              <span
                aria-disabled={false}
                className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-176 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={null}
              >
                <span
                  className="MuiIconButton-label"
                >
                  <input
                    className="PrivateSwitchBase-input-179"
                    data-indeterminate={false}
                    onChange={[Function]}
                    type="checkbox"
                    value="sudeste"
                  />
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                </span>
              </span>
              <span
                className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                Sudeste
              </span>
            </label>
            <label
              className="MuiFormControlLabel-root"
            >
              <span
                aria-disabled={false}
                className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-176 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={null}
              >
                <span
                  className="MuiIconButton-label"
                >
                  <input
                    className="PrivateSwitchBase-input-179"
                    data-indeterminate={false}
                    onChange={[Function]}
                    type="checkbox"
                    value="sul"
                  />
                  <svg
                    aria-hidden="true"
                    className="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                </span>
              </span>
              <span
                className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                Sul
              </span>
            </label>
          </div>
        </article>
      </div>
    </div>
  </div>
</div>
`;
